coreos.sh

```diff
@@ -246,15 +245,15 @@ add_coreos_oem_scripts() {
     local scriptpath="$mntpath/bin"
     local scriptfile="$scriptpath/netname.sh"
     if [ ! -d "$scriptpath" ]; then
-      mkdir -p $scriptpath
+      mkdir -p "$scriptpath"
     fi
-    cat << EOF >> $scriptfile
+    cat << EOF >> "$scriptfile"
```

Comment:
we should migrate all heredoc notations to printf in this file, I will create a seperate issue for this